ui/admin/app/routes/scopes/scope/workers/worker/create-tags.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable ember/no-controller-access-in-routes */
+import Route from '@ember/routing/route';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { TrackedArray } from 'tracked-built-ins';
+
+export default class ScopesScopeWorkersWorkerCreateTagsRoute extends Route {
+  // =services
+
+  @service confirm;
+  @service intl;
+
+  @action
+  async willTransition(transition) {
+    if (this.controller.get('apiTags').length) {
```

Comment:
I think if we access the controller in a route, we're supposed to still call `controllerFor` according to the [rule](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-controller-access-in-routes.md)

Reply:
Sorry, not sure why I changed that. I used controllerFor in my first interation.